packages/core-plugin-api/src/plugin/types.ts

```diff
@@ -74,6 +75,62 @@ export type PluginFeatureFlagConfig = {
   name: string;
 };
 
+export type PluginInfoLink = {
+  /**
+   * The url to the external site, document, etc.
+   */
+  url: string;
+
+  /**
+   * An optional descriptive title for the link.
+   */
+  title?: string;
+};
+
+/**
+ * PluginInfo
+ *
+ * @public
+ */
+export type PluginInfo = {
+  packageJson?: unknown;
+
+  /**
+   * Plugin name; defaults to `backstage.name` in package.json
+   */
+  name?: string;
```

Comment:
There is already a plugin ID, so feeling this would be unfortunate. Perhaps a lot of this can be bundled up in some kind of "package metadata" instead? I think overall it would perhaps be good to group these properties up a bit, perhaps one metadata chunk, and another ownership & support chunk. Decoration form the app would then only be done of ownership & support

Reply:
Would `title` be better?

Reply:
Resolved by removing `name`